programs/ssl/ssl_client2.c

```diff
@@ -4040,6 +4040,11 @@ int main( int argc, char *argv[] )
         }
 #endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_ZERO_RTT)
```

Comment:
It turns out that the test succeeds even if you remove this: The reason is that in #261 we should have cleared the 0-RTT configuration as part of `mbedtls_ssl_session_reset()`. Could you do this in this PR, @zhihan?